swarm/storage/mru/signedupdate_test.go

```diff
@@ -0,0 +1,142 @@
+package mru
```

Comment:
This was all moved from somewhere?

Reply:
Yes, sorry. Just moved it off `resource.go` which traditionally had most of the testing code.

Reply:
It has changes too, I presume? :) (since it's a test, it's not so important)

Reply:
A few touches. Not worth your time ;)